pyproject.toml

```diff
@@ -166,6 +167,7 @@ all_extras_pandas2 = [
     "scikit_posthocs>=0.6.5",
     "seaborn>=0.11.0",
     "seasonal",
+    "skpro<2.1.0",
```

Comment:
Same as above, but here quite a few doesn't have it so deprioritised (?)

Reply:
done